src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,447 @@
+| InjectionEnd    | RECOMMENDED           | [number][]               | Time of end of injection with respect to `TimeZero` in the default unit seconds.                                                                                          |
+
+#### Reconstruction
+
+This section is mandatory and contains information about the image reconstruction.
+
+All reconstruction specific parameters that are not specified, but one wants to include, should go into the `ReconMethodParameterValues` field.
+
+| **Key name**                         | **Requirement level** | **Data type**                          | **Description**                                                                                                 |
+| ------------------------------------ | --------------------- | -------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| AcquisitionMode                      | REQUIRED              | [string][]                             | Type of acquisition of the PET data (for example, `"list mode"`)                                                |
+| ImageDecayCorrected                  | REQUIRED              | [boolean][]                            | Boolean flag specifying whether the image data have been decay-corrected.                                       |
+| ImageDecayCorrectionTime             | REQUIRED              | [number][]                             | Point in time from which the decay correction was applied with respect to TimeZero in the default unit seconds. |
+| ReconMethodName                      | REQUIRED              | [string][]                             | Reconstruction method or algorithm (for example, `"3d-op-osem"`).                                               |
+| ReconMethodParameterLabels           | REQUIRED              | [array][] of [strings][]               | Names of reconstruction parameters (for example, `["subsets", "iterations"]`)                                   |
+| ReconMethodParameterUnit             | REQUIRED              | [array][] of [strings][]               | Unit of reconstruction parameters (for example, `["none", "none"]`).                                            |
+| ReconMethodParameterValues           | REQUIRED              | [array][] of [numbers][]               | Values of reconstruction parameters (for example, `[21, 3]`)                                                    |
+| ReconFilterType                      | REQUIRED              | [string][] or [array][] of [strings][] | Type of post-recon smoothing (for example, `["Shepp"]`)                                                         |
+| ReconFilterSize                      | REQUIRED              | [number][] or [array][] of [numbers][] | Kernel size of post-recon filter (FWHM).                                                                        |
```

Comment:
Could you write an example? how do you express the kernel size?
I've seen the following in pet_example3:

        "ReconFilterType": ["Image filter were Shepp 0.5 (2.5 mm in FWHM)","the Z filter was all-pass 0.4 (2.0 mm in FWHM)"],
        "ReconFilterSize": [2.5, 2.0],

I was a bit surprised since I wasn't expecting that much text description inside the field ReconFilterType

Reply:
This is example data to mimic Parsey et al. 2006 (http://jnm.snmjournals.org/cgi/content/abstract/47/11/1796%5Cnhttp://jnm.snmjournals.org/cgi/content/full/47/11/1796). 

Here they state the following in the text: "The reconstruction filter and estimated image filter were Shepp 0.5 (2.5 mm in full width at half maximum), the Z filter was all-pass 0.4 (2.0 mm in full width at half maximum), and the zoom factor was 4.0, leading to a final image resolution of 5.1 mm in full width at half maximum at the center of the field of view (25).". 

I agree with your comment, that it is a fairly long text description, but not sure how to circumvent this to make it more clear. In another example dataset (https://www.dropbox.com/sh/2vw8daz1j661mvp/AAClJnc49e52wnvF6-YHUuWIa?dl=0), it is simply a Gaussian 3D filter.

Reply:
Yes, these fields are not clear. 

- Could more than one filter be applied? Are they applied to different things?

- In the example3, there's some redundant information, e.g. "(2.5 mm in FWHM)" in ReconFilterType, when you also have it in ReconFilterSize.

- It's not clear if the ReconFilterSize is [2.5, 2.0], how would you differentiate if it's from two filters, or if its from just one, like in the example you send (AZ_visual_stimuli):

        "ReconFilterType": "Gaussian",
	"ReconFilterSize": [4,4,4],

- Also you could specify the units in the field (it seems to be mm)

Reply:
Good point. 

1) Yes, more than one filter can be applied. However, usually only one filter is applied.
2) I agree there is redundant information in the example data set. I have updated the example data set.
3) I would argue, that in most cases a isotropic filter is applied, and hence each entry in the FilterSize should correspond to an entry in the FilterType. Therefore, the AZ_visual_stimuli should only have one entry (I have already made this update). I think this should be checked in the validator, that ReconFilterType and ReconFilterSize have the same number of entries. 
4) You are right. This should be specified. 

```suggestion
| ReconFilterSize                      | REQUIRED              | [number][] or [array][] of [numbers][] | Kernel size of post-recon filter (FWHM) in default units `mm`.                                                                        |
```